### DISCUSSION TITLE
Tijd en Object Status modelleren

### DISCUSSION THREAD

**Discussion by lvdbrink** [Geonovum/disgeo-imsor#27] [General]
Michel: 

Zie bijlage:
[TemporeelModelleren.pptx](https://github.com/Geonovum/disgeo-imsor/files/6205998/TemporeelModelleren.pptx)

> Nav discussie vandaag m’n eigen gedacht geprobeerd te ordenen…
> 
> Misschien al allemaal aannamen voor sommigen of juist onjuist.
> 
> Is iig hoe ik er nu tegenaan kijk.
>

**Comment by RiX012** [comment]
Het lijken mij zo elke keer 'uit de lucht gegrepen' indelingen. De atomaire (naar mijn mening) opsplitsing die Michel doet is goed. Maar is het niet beter/mogelijk om aan te sluiten bij standaarden die deze "waardelijsten" definiëren. Volgens mij zijn er genoeg ISO normen die hier wat over zeggen (ik ken alleen de ISO15288 goed).

**Reply by lvdbrink** [reply]
Ik ken hier geen standaarden voor, maar suggesties zijn welkom. @JvGldr misschien weet jij iets? Een Asset lifecycle management standaard?

**Reply by RiX012** [reply]
De ISO55000 wordt uiteraard veel aangehaald.
https://formaspace.com/wp-content/uploads/2013/11/PAS-55-addresses-asset-lifecycle-management1HR.jpg

**Reply by JvGldr** [reply]
Ja, ISO55000 is een bekende. PAS-55 ook, en er is ook een recentere en dat is NEN-ISO 55001. Er zijn ook nog een paar de facto ALCM standaarden. Zo heeft SAP een EAM (Enterprise Asset Management) module, waar ALCM in zit. Maar allemaal zijn het de bekende cirkels, waarbij de belangrijkste smaken zijn: Cradle to Cradle en Cradle to Grave. De stadia in de cycle zijn soms meer en soms minder gedetailleerd. De planfase kan je ook nog weer opdelen in verschillende onderliggende planfasen (van houtskoolschets tot een plan in AutoCad o.i.d). De data die je vastlegt wordt dan gaandeweg de fasen ook gedetaillleerder. het idee is wel dat je de data "meeneemt" gedurende die verschillende fasen.

**Comment by RuudKathmann** [comment]
Welke statussen worden onderscheiden lijkt ons typisch een onderwerp voor de werkgroep inhoud.
Die hebben daarvoor een voorstel gedaan en dat ligt nu voor ter consultatie.
Bij de modellering moeten wij gewoon uitvoering geven aan deze specificaties.
Natuurlijk zijn er in het ontwerpproces meer details mogelijk.
Maar de SOR is niet de ontwerpomgeving waar planning in ontworpen worden.
Toekomstige objecten (planobjecten) worden alleen in de SOR opgenomen om te kunnen dienen als referentie voor verdere ontwikkelingen in de omgeving.
De globale planfasen zijn daarbij voldoende.
Wanneer alle planfasen in de SOR bijgehouden zouden moeten worden, is de SOR niet bij te houden.
Ieder architect die bezig is met het ontwerp en detaillering van een gebouw zou dan continu zijn nieuwe versie in de SOR moeten laten verwerken.
Er moeten vanuit de informatiemodellering wel heel goede argumenten komen om de bij de inhoudelijke discussie gekozen statussen aan te passen.
Vooralsnog gaan wij ervan uit dat bij de modellering exact gemodelleerd wordt, wat er beschreven staat in het conceptueel model.

**Reply by RiX012** [reply]
> Vooralsnog gaan wij ervan uit dat bij de modellering exact gemodelleerd wordt, wat er beschreven staat in het conceptueel model.

Goed punt, misschien moeten we daar aangeven dat ze geen dingen zelf moeten gaan verzinnen.

**Reply by lvdbrink** [reply]
Voor de duidelijkheid: het document van de werkgroep Inhoud beschrijft de "eisen aan het model van de SOR" (ik noem het document altijd "EMSO". Het is geen conceptueel model, al wordt het in de wandelgangen helaas wel vaak zo genoemd. 

Het team dat het informatiemodel voor de SOR maakt, met de expertgroep die hiervoor is opgericht, vertaalt deze inhoudelijke eisen naar een informatiemodel. Daarbij nemen wij geen beslissingen over de inhoud, maar wel over hoe we die in het informatiemodel uitdrukken, met daarbij ook de beleidsvisie over DisGeo en de architectuurbeschrijving in ons achterhoofd. Het EMSO is een document dat niet "exact", i.e. formeel, maar in natuurlijke taal beschrijft wat de inhoud van de SOR moet worden. Dat moeten we dus als informatiemodelleurs interpreteren. Daarbij denken we soms ook na over de inhoud (hoe wordt wat er staat precies bedoelt?). Ook gaat het EMSO hier en daar iets te ver richting informatiemodelleren: dan beschrijft het al een oplossing, in plaats van de inhoudelijke eis. We maken daarin dan voor het informatiemodel onze eigen keuzes, die we toetsen bij de EMSO auteurs. 

Het is uiteraard belangrijk om erop te letten dat we niet gaan tornen aan inhoudelijke eisen, of discussies over de inhoud opnieuw gaan voeren.
